src/video.py

```diff
@@ -2,23 +2,24 @@
 
 import os
 import uuid
-from .db import delete_video_db, get_video_list_db, save_video_db
+from .db import delete_video_db, get_video_list_db, get_user_id_from_video, save_video_db
 from .user import get_user_id
 
-
 class Video():
     """Video object containing ID, extension, and uploader."""
 
     def __init__(self):
         self.video_id = ''
         self.extension = ''
         self.user_id = ''
+        self.filename = ''
```

Comment:
I don't think this field is used anywhere. Remove the line?

I don't think we want to have this field available since it's the unsafe filename and not the UUID filename.

Reply:
agreed.